src/api_routes_v4.rs

```diff
@@ -0,0 +1,383 @@
+          .route("", get().to(get_comment))
+          .route("", put().to(update_comment))
+          .route("/delete", post().to(delete_comment))
+          .route("/remove", post().to(remove_comment))
+          .route("/mark_as_read", post().to(mark_reply_as_read))
+          .route("/distinguish", post().to(distinguish_comment))
+          .route("/like", post().to(like_comment))
+          .route("/like/list", get().to(list_comment_likes))
+          .route("/save", put().to(save_comment))
+          .route("/list", get().to(list_comments))
+          .route("/report", post().to(create_comment_report))
+          .route("/report/resolve", put().to(resolve_comment_report))
+          .route("/report/list", get().to(list_comment_reports)),
+      )
+      // Private Message
+      .service(
+        scope("/private_message")
+          .route("/list", get().to(get_private_message))
+          .route("", post().to(create_private_message))
+          .route("", put().to(update_private_message))
+          .route("/delete", post().to(delete_private_message))
+          .route("/mark_as_read", post().to(mark_pm_as_read))
+          .route("/report", post().to(create_pm_report))
+          .route("/report/resolve", put().to(resolve_pm_report))
+          .route("/report/list", get().to(list_pm_reports)),
+      )
+      // User
+      .service(
+        scope("/account/auth")
+          .guard(guard::Post())
+          .wrap(rate_limit.register())
+          .route("/register", post().to(register))
+          .route("/login", post().to(login))
+          .route("/logout", post().to(logout))
+          .route("/password_reset", post().to(reset_password))
+          .route("/get_captcha", get().to(get_captcha))
+          .route("/password_change", post().to(change_password_after_reset))
+          .route("/change_password", put().to(change_password))
+          .route("/totp/generate", post().to(generate_totp_secret))
+          .route("/totp/update", post().to(update_totp))
+          .route("/verify_email", post().to(verify_email)),
+      )
+      .service(
+        scope("/account/settings")
+          .wrap(rate_limit.import_user_settings())
+          .route("/export", get().to(export_settings))
+          .route("/import", post().to(import_settings)),
+      )
+      .service(
+        scope("/account")
+          .route("/my_user", get().to(get_my_user))
+          .route("/list_media", get().to(list_media))
+          .route("/mention", get().to(list_mentions))
+          .route("/replies", get().to(list_replies))
+          .route("/block", post().to(block_person))
+          .route("/delete", post().to(delete_account))
+          .route(
+            "/mention/mark_as_read",
+            post().to(mark_person_mention_as_read),
+          )
+          .route(
+            "/mention/mark_all_as_read",
+            post().to(mark_all_notifications_read),
+          )
+          .route("/settings/save", put().to(save_user_settings))
```

Comment:
I still think this should just be `/save`, but all of this needs feedback from others also.

Reply:
`/account/save`? Its not saving the account but the settings.